runner/src/server/lib/consoleMessages/welcome.ts

```diff
@@ -0,0 +1,36 @@
+import config from "server/config";
+
+function welcome() {
```

Comment:
I think all of these messages are best suited in buildConfig()?

Reply:
buildConfig() doesn't have server (or request), did you want these messages loged in the same way? If so what's the best way to do that?